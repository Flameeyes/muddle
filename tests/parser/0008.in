ifneq?

Fish ${fn:ifneq("/top/fish","soup")

Fish is not soup!
}

Soup: ${fn:ifneq("/tmp/fish","fish")
Fish is not fish!
}
