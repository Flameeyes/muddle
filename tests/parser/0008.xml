<?xml version="1.0" ?>

<top>
  <fish>soup</fish>
</top>

