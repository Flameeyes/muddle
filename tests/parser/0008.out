ifneq?

Fish 

Soup: 
Fish is not fish!

